lib/vector-api-client/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "vector-api-client"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Not sure if this version bump is meaningful. I think our consensus for internally used crates is that we don't care about versions yet, as we assume they're only used within our "tiny monorepo".
This should probably be given some thought.